pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -67,6 +67,11 @@ func (d *ProjectStatusDescriber) MakeGraph(namespace string) (osgraph.Graph, uti
 	if errs := parallel.Run(loadingFuncs...); len(errs) > 0 {
 		actualErrors := []error{}
 		for _, err := range errs {
+			// ignore not found errors for resources which may be disabled server side
+			// this check should be more specific but nothing is reported in the details.
+			if kapierrors.IsNotFound(err) {
+				continue
```

Comment:
I think the *correct* thing to do would be to skip adding the buildLoader and bcLoader to the list of loading functions if builds weren't enabled on the server. For now, you move this check into the buildLoader/bcLoader to tolerate NotFound errors on those two types in particular, with a TODO to check swagger for feature enablement